setup/send_assets_to_multi_giveaway.ts

```diff
@@ -61,18 +61,12 @@ const func: DeployFunction = async function () {
   const ids = [];
   const values = [];
   for (const assetId in assetIdsCount) {
-    const balance: BigNumber = await read(
```

Comment:
TODO: add flag argument to calculate diff (it's useful when a claim was setup wrongly and there are some assets in the contract already)